### PR TITLE
Added validations to exercises types

### PR DESCRIPTION
**Pull request by fedescarpa**

### CODE REVIEW

File: src/scripts/config/strings/lang-en.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ angular
     error_editor_single_choice_validation_only_one: '{{fullName}}: Shoud have only one choice selected',
     error_expectations_incomplete_validation: '{{fullName}}: Has incomplete expectations',
     error_mandatory_field: '{{ fullName }}: Field {{ field }} is mandatory',
+    error_queriable_language_validation: '{{ exercise }}: Language {{ language }} does not supports playground exercises',
+    error_triable_language_validation: '{{ exercise }}: Language {{ language }} does not supports interactive exercises',
```
Comment (flbulgarelli) [inline src/scripts/config/strings/lang-en.js]: Support

File: src/scripts/config/strings/lang-es.js [hunk 2]
```diff
@@ -41,6 +41,8 @@ angular
     error_editor_single_choice_validation_only_one: '{{fullName}}: Debe seleccionar solamente una opción',
     error_expectations_incomplete_validation: '{{fullName}}: Tiene expectativas incompletas',
     error_mandatory_field: '{{fullName}}: El campo "{{ field }}" no puede estar vacío',
+    error_queriable_language_validation: '{{ exercise }}: El lenguage "{{ language }}" no soporta ejercicios de tipo juego',
+    error_triable_language_validation: '{{ exercise }}: El lenguage "{{ language }}" no soporta ejercicios interactivos',
```
Comment (flbulgarelli) [inline src/scripts/config/strings/lang-es.js]: Why the "?

Comment (fedescarpa) [reply src/scripts/config/strings/lang-es.js]: Because I'm so stupid

File: src/scripts/config/strings/lang-en.js (hunk 1 above)
Comment (fedescarpa) [reply src/scripts/config/strings/lang-en.js]: Done

**Pull request merged**